### ISSUE TITLE
feat: Open note without `data`?

### ISSUE THREAD

**Issue by ankushg** [Vinzent03/obsidian-advanced-uri#9]
It would be really nice if specifying a `filepath` but no `data` would just open the given path.

I'm building some iOS Shortcuts to automate some flows, and I'd love to rely only on `obsidian-advanced-uri` instead of switching between this and the official `open` action depending on whether or not I have data

Right now it looks like this case is ignored by `registerObsidianProtocolHandler`, when it can probably just be implemented with `this.app.workspace.openLinkText(parameters.filepath, "");`

A nice-to-have would be if the cursor were at the appropriate position based on `mode=append` or `mode=prepend`

**Comment by Vinzent03** [comment]
Yeah I didn't add it, because it's covered by the official `open` action, but I see your wish to stay with one action. I like the idea with `mode=append` and `mode=prepend`.
